schemas/blocks/unrelated_relationship_block.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://eq.ons.gov.uk/blocks/unrelated_relationship_block.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["UnrelatedRelationship"]
```

Comment:
As distinct from related relationships? I think `UnrelatedQuestion` is better, and more consistent with the list question sub blocks.